rke/k8s_rke_system_images.go

```diff
@@ -5690,6 +5733,49 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in v2.5.11
```

Comment:
The comment needs to be `Enabled out of band post v2.5.10` because we're releasing these versions out of band before v2.5.11 gets released. Same for v1.19.16.

Reply:
Okay..got it! I updated the comments.